pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	stdError "errors"
 	"fmt"
+	"k8s.io/client-go/kubernetes"
 	"time"
```

Comment:
```suggestion
	"context"
	stdError "errors"
	"fmt"
	"time"

	"k8s.io/client-go/kubernetes"
```
How `go fmt` dint catch this :thinking: